urllib3/exceptions.py

```diff
@@ -144,6 +144,11 @@ class ResponseError(HTTPError):
     SPECIFIC_ERROR = 'too many {status_code} error responses'
 
 
+class SSLConfigurationWarning(HTTPWarning):
```

Comment:
Should this be a SecurityWarning? (See exceptions below)

Also, maybe call it `InsecurePlatformWarning`?